info/diktat-kotlin-coding-style-guide-en.md

```diff
@@ -1091,6 +1091,7 @@ fun baz() {
 
   - Two colons (`::`) are written without spaces: `Object::toString`
   - Dot separator (`.`) that stays on the same line with an object name `object.toString()`
+  - Operator `..` for creating ranges, e.g. `1..100`
```

Comment:
compilation does not prohibit anything else?

Reply:
I have checked, things like `1 .. 100` are allowed, you can even insert newline after the `..`.